Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM alpine:3.10
+FROM python:3.9
 
-COPY LICENSE README.md entrypoint.sh /
+WORKDIR /opt/action
 
-COPY src/create-envfile.py /create-envfile.py
+COPY LICENSE README.md ./
 
-RUN apk add python3
+COPY src/create-envfile.py ./create-envfile.py
 
-ENTRYPOINT ["/entrypoint.sh"]
+ENTRYPOINT ["./opt/action/create-envfile.py"]
```

Comment:
Shouldn't this be an absolute path?

Reply:
my mistake, i think worth doing `./create-envfile.py`

Reply:
my mistake, i fixed at new commit